sdk/vppagent/dataplane/chained_dataplane_server.go

```diff
@@ -0,0 +1,45 @@
+package dataplane
+
+import (
+	"context"
+
+	"github.com/golang/protobuf/ptypes/empty"
+
+	"github.com/networkservicemesh/networkservicemesh/controlplane/pkg/apis/crossconnect"
+	"github.com/networkservicemesh/networkservicemesh/dataplane/pkg/apis/dataplane"
+	"github.com/networkservicemesh/networkservicemesh/sdk/vppagent/dataplane/state"
+)
+
+type ChainedDataplaneServer interface {
```

Comment:
Lets just use dataplane.DataplaneServer rather than wrapping in another interface.

Reply:
thanks, fixed